frontend/src/static/css/style.scss

```diff
@@ -408,6 +409,14 @@ header{
                     margin-left:0.25rem;
                 }
             }
+
+            .reverse {
+                color:mui-color("black");
+            }
+
+            select {
+                font-size:0.875rem;
```

Comment:
instead of doing this, perhaps specify our own font class and assign it to the elements?